### PR TITLE
Translating work to a sifter based transform

### PR DESCRIPTION
**Pull request by kellrott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,48 +25,44 @@ Load your environment:
...
-  table      Extract project summary.
+go install github.com/bmeg/sifter@latest
+go install github.com/bmeg/lathe@latest
+```
 
+### Update path
 
+```
+export PATH=$PATH:$HOME/go/bin
 ```
```
Comment (bwalsh) [inline README.md:45]: We should add ### Install snakemake

```
brew install snakemake
```

Comment (bwalsh) [reply README.md:45]: Also saw a typo in my instructions.


> pip intall -r requirements.txt
> pip intall -e .

Should be install

**Review by bwalsh — COMMENTED** [review]
Minor suggestions to the README below

I downloaded ran locally, no problems

I'll need to clean up the data model repo if we are going to be using it as a submodule

**Comment by bwalsh** [comment]
LGTM
+1

File: README.md (hunk 1 above)
Comment (kellrott) [reply README.md:45]: I tried adding snakemake to the requirements, so it would pip install.

Comment (matthewpeterkort) [reply README.md:45]: had to manually "pip install snakemake" even though it was in requirements.txt. After that all commands from the README.md ran successfully but there is no outputs directory in root.

Comment (matthewpeterkort) [reply README.md:45]: Screen Shot 2023-02-10 at 10 53 01 AM

**Review by matthewpeterkort — APPROVED** [review]

**Pull request merged**